picoquic/tls_api.c

```diff
@@ -660,6 +660,22 @@ void picoquic_master_tlscontext_free(picoquic_quic_t* quic)
     }
 }
 
+/*
+ * Get the same time simulation as used for TLS 
+ */
+
+uint64_t picoquic_get_tls_time(picoquic_quic_t* quic)
+{
+    uint64_t now;
+    if (quic->p_simulated_time == NULL) {
+        now = picoquic_current_time();
```

Comment:
Hmm, in picoquic-rs I do not use the `picoquic_current_time` function. My timer starts when I create the initial context and counts upwards. So, I don't have all the milliseconds since 1970. If we start using the timer function internally, I would propose to remove giving the current time into the public interface.